config/custom-environment-variables.yaml

```diff
@@ -64,8 +64,9 @@ datastore:
         prefix: DATASTORE_SEQUELIZE_PREFIX
 
 executor:
-    plugin: EXECUTOR_PLUGIN
```

Comment:
There's Petey's suggestion and there's St. John's suggestion. I went with Petey's. Unless we think I should go with St. John's?